src/Dto/StimulusControllersDto.php

```diff
@@ -36,10 +39,21 @@ public function addController(string $controllerName, array $controllerValues =
         foreach ($controllerClasses as $key => $class) {
             $key = $this->escapeAsHtmlAttr($this->normalizeKeyName($key));
 
-            $this->values['data-'.$controllerName.'-'.$key.'-class'] = $class;
+            $this->classes['data-'.$controllerName.'-'.$key.'-class'] = $class;
         }
     }
 
+    public function addOutlet(string $outletName, string $selector)
+    {
+        if (1 < \count($this->controllers)) {
+            throw new \LengthException('You cannot call addOutlet() method when passing more than one controller identifier to stimulus_controller() function');
+        }
+
+        $this->outlets['data-'.$this->controllers[0].'-'.$outletName.'-outlet'] = $selector;
+
+        return new Markup($this, 'UTF-8');
```

Comment:
Why is the `Markup` class suddenly needed in this situation?

Reply:
I'm probably doing something wrong but if I don't use this class, the result becomes : `data-controller=""controller"" data-controller-foo-outlet="".foo""`

Reply:
Weird. It make me think that, below in `__toString()`, when we call `$this->escapeAsHtmlAttr($value)`, that is taking `.foo` and turning it into `".foo"`... but that seems very odd...

Reply:
It's also clean when I use the raw filter like this : {{ stimulus_controller('foo').addOutlet('bar', '.bar')|raw }}, but it's certainly not what we need.